uguid/tests/test_guid.rs

```diff
@@ -69,6 +69,15 @@ fn test_guid() {
     assert_eq!(guid!("01234567-89ab-cdef-0123-456789abcdef"), guid);
 }
 
+#[test]
+fn test_from_random_bytes() {
+    let guid = Guid::from_random_bytes([
```

Comment:
It might not be obvious to a reader that this set of bytes was chosen so `Guid::from_random_bytes(...)` != `Guid::from_bytes(...)`.
Is it worthwhile/in scope for the test to clearly show this (comment or an additional couple of asserts)?


Is it worth confirming that the only difference in the GUID is the variant/version between from_random_bytes and from_bytes?

Reply:
Good call, updated to check the full set of bytes.